docs/compilers/CSharp/Compiler Breaking Changes - post VS2019.md

```diff
@@ -8,3 +8,5 @@ Each entry should include a short description of the break, followed by either a
     bool M<T>(T t) => t is null;
     ```
 However, in *Visual Studio 2019* we improperly permitted this to compile in language versions `7.0`, `7.1`, `7.2`, and `7.3`.  In *Visual Studio 2019 Update 1* we will make it an error (as it was in *Visual Studio 2017*), and suggest updating to `preview` or `8.0`.
+
+2. https://github.com/dotnet/roslyn/issues/37527 The constant folding behavior of the compiler differed depending on your host architecture when converting a floating-point constant to an integral type where that conversion would be a compile-time error if not in an `unchecked` context.  We now yield a zero result for such conversions on all host architectures.
```

Comment:
For additional reference, from IEEE 754:2008 (the same wording exists in the new 754:2019 spec, but the entry is `ii`, rather than `i`):
> For operations producing no result in floating-point format, the operations that signal the invalid operation exception are:
>
> i) conversion of a floating-point number to an integer format, when the source is NaN, infinity, or a value that would convert to an integer outside the range of the result format under the applicable rounding attribute

This leaves the result as undefined. The C and C++ language specifications do not explicitly define a result for these either.